### PR TITLE
[noTicket][risk=no]Fix: Workspace About page just showing Research purpose options

### PR DESCRIPTION
**Pull request by NehaBroad**
Screen Shot 2020-04-01 at 10 07 08 PM
Follow up ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-4694
---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/workspace/research-purpose.tsx [hunk 1]
```diff
@@ -57,7 +57,8 @@ const styles = reactStyles({
 export const ResearchPurpose = withCurrentWorkspace()(
   ({workspace}: {workspace: WorkspaceData}) => {
     const isOwner = WorkspacePermissionsUtil.isOwner(workspace.accessLevel);
-    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose);
+    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, true);
+    const selectedPrimaryPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, false);
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/research-purpose.tsx:61]: open for name suggestion: selectedPrimaryPurpose seems more like all the primary purpose but the name should reflect  all Primary Purpose except Research Purpose

Comment (NehaBroad) [inline ui/src/app/pages/workspace/research-purpose.tsx:61]: open for name suggestions : This should reflect all selected Primary purpose  that are not research purpose

Comment (blrubenstein) [reply ui/src/app/pages/workspace/research-purpose.tsx:61]: The name seems fine to me.

File: ui/src/app/pages/workspace/workspace-about.spec.tsx [hunk 2]
```diff
@@ -63,6 +63,10 @@ describe('WorkspaceAbout', () => {
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper.exists('[data-test-id="researchPurpose"]'))
       .toBeTruthy();
+    // Research Purpose: Drug, Population and Ethics
+    expect(wrapper.find('[data-test-id="primaryResearchPurpose"]').length).toBe(3);
+    // Primary Purpose: Eduction
```
Comment (blrubenstein) [inline ui/src/app/pages/workspace/workspace-about.spec.tsx:68]: Nit: Spelling (education)

**Comment by NehaBroad** [comment]
Yes since research purpose categories are just one of the many options and not all (primary purpose also has other, profile and education)

**Pull request merged**